src/calendar/styles.scss

```diff
@@ -200,8 +201,6 @@
       }
     }
   }
-
-  @include styles.styles-reset;
```

Comment:
I do not know what to say here. Hiding the reset line here is a readability issue, great that the new deprecation forces us to fix this